cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -73,7 +82,10 @@ void createTableIfNotExists(TableDescriptor descriptor, @Nullable byte[][] split
   void enableTableIfDisabled(String namespace, String name) throws IOException;
 
   /**
-   * Disable the specified table if it is enabled.
+   * Disable the specified table if it is enabled. Changing the dataset specs results into
```

Comment:
This is called when an HBase backed dataset has its properties modified. In order to modify the HBase table, CDAP first disables it with this method, then calls modifyTable, then calls enableTableIfDisabled.